### DISCUSSION TITLE
Changelog please!

### DISCUSSION THREAD

**Discussion by saranglakare** [k3nsei/ng-in-viewport#1282] [General]
First of all, thanks for the developers for this wonderful library. I noticed that a new version was released (`15.0.0`) a few days ago. However, it is not clear what changed vs the last version (`13.0.1`). I would kindly request devs to update the Changelog (`https://github.com/k3nsei/ng-in-viewport/blob/develop/CHANGELOG.md`) - it would it really easy for users to update to the latest version knowing what the changes are.

Thanks a ton!

**Comment by k3nsei** [comment]
@saranglakare changes are described in releases page https://github.com/k3nsei/ng-in-viewport/releases

**Reply by saranglakare** [reply]
Thank you very much! Could you please add this link in the `CHANGELOG.md` file? That's usually the go to place to check for the change log. Also, considering you do have the prior version there, so gives an "incomplete" feeling to it. Thanks!

**Reply by k3nsei** [reply]
@saranglakare Feel free to create PR with contribution from your side. I could delete CHANGELOG.md from repo as quick fix. I have my own life and nobody is paying me for those changes. I lost almost 3 weekends on rewrite. Even if there is sponsors program enabled on this repo and so many people uses that lib nobody even bought me a beer or coffee.

**Reply by saranglakare** [reply]
Yes, I understand.. been there, done that! What this work mainly brings is visibility to you. It is highly appreciated. Such contributions over time is what is taking us all forward.. one step at a time! So thanks.

Will definitely try to send a PR.

**Reply by k3nsei** [reply]
@saranglakare I know your point of view. But for a few days I need a brake, because as I said before, last rewrite took a lot of time and energy from me. It's not that I don't care about this lib. I even automated generation of changelog by usage of https://github.com/google-github-actions/release-please-action . But this will have effect on future releases.  If you want to help, I'd appreciate a PR.

**Comment by k3nsei** [comment]
Changelog has been updated
